_layouts/categories.html

```diff
@@ -0,0 +1,36 @@
+---
+layout: base
```

Comment:
```suggestion
layout: default
```

Reply:
I used base layout because i didnt want section and div properties 
it affects the css behavior of the page

so i copied the initial function of the default layout into the categories layout

Reply:
> I used base layout because i didnt want section and div properties
it affects the css behavior of the page

You used the section and div. I am not sure to understand your thoughts.

Anyway, the behavior should be the same as in the posts.html page, which uses the default layout. Otherwise, it makes a page different from the others.